web/modules/custom/itk_cookie_message/js/itk-cookie-message.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file
+ * Enables the display of the EU cookie dialog.
+ */
+
+(function ($, Drupal) {
+  'use strict';
+
+  Drupal.behaviors.cookieMessageBehavior = {
+    attach: function (context, settings) {
+      var cookieName = drupalSettings.itk_cookie_message.cookie_name;
+
+      // Inline function to get the current value of the cookie.
+      var cookieValue = function () {
+        var regex = new RegExp('(?:^|; )' + encodeURIComponent(cookieName) + '=([^;]*)');
+        var result = regex.exec(document.cookie);
+        return result ? (result[1]) : null;
```

Comment:
result[1] should that be result.length = 1 ?

And should it return result[1] and not result?